src/app/webbrowser/SettingsPage.qml

```diff
@@ -86,6 +86,24 @@ FocusScope {
                     ListItemLayout {
                         title.text: i18n.tr("Homepage")
                         subtitle.text: homepageListItem.currentHomepage
+                        Icon {
+                            id: resetHomepage
+                            name: "reset"
+
+                            height: units.gu(2)
+                            width: height
+                            opacity: (settingsObject.homepage.toString() === settingsObject.getDefaultHomepage()) ? 0.5 : 1
```

Comment:
- Two questions:
 1) Isn't possible to use `color: theme.palette.disabled.backgroundText`?
 2) Does the MouseArea need to be disabled? I know would do nothing the second time but still